sale_order_report_without_price/report/sale_report_templates.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="report_saleorder_document_without_price"
+        inherit_id="sale.report_saleorder_document"
+        primary="True"
+        priority="99"
+    >
+        <xpath expr="//th[@name='th_priceunit']" position="replace" />
```

Comment:
@marielejeune Wasn't it possible to avoid replaces ?

Reply:
Indeed, adding `invisible="1"` is better. Done